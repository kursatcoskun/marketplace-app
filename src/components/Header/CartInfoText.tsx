import Text, { TextProps } from "antd/lib/typography/Text";
import React from "react";
import styled from "styled-components";

export const CartInfoText: React.FunctionComponent<TextProps> = styled(Text)`
  color: white;
  margin-left: 3px;
`;
