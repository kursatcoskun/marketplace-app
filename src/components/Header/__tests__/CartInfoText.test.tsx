// MyComponent.test.js
import React from "react";
import { shallow } from "enzyme";
import { CartInfoText } from "../CartInfoText";
describe("CartInfoTextComponent", () => {
  it("should render CartInfoText component", () => {
    const wrapper = shallow(<CartInfoText />);
  });

  it("should render initial layout", () => {
    const component = shallow(<CartInfoText />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
