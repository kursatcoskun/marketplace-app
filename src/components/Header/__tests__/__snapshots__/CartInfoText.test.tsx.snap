// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartInfoTextComponent should render initial layout 1`] = `
Array [
  <Text
    className="sc-bdnxRM jmGxms"
  />,
]
`;
